sapmachine/content.md

```diff
@@ -0,0 +1,35 @@
+### Overview
+
+The image in this repository contains the SapMachine Java virtual machine (JVM). SapMachine is an OpenJDK based JVM that is build, quality tested and long-term supported by SAP. It is the default JVM on the [SAP Cloud Platform](https://cloudplatform.sap.com/index.html) and it is also supported as a [Standard JRE](https://github.com/cloudfoundry/java-buildpack/blob/master/docs/jre-sap_machine_jre.md) in the [Cloud Foundry Java Build Pack](https://github.com/cloudfoundry/java-buildpack).
+
+For more information see the [SapMachine website](https://sapmachine.io).
+
+The SapMachine image supports the x86/64 architecture.
+
+Java and all Java-based trademarks and logos are trademarks or registered trademarks of Oracle and/or its affiliates.
+
+
```

Comment:
I think the double blank line here is probably why Travis is complaining.

(Instead of running "check-markdownfmt" you probably want to run `./markdownfmt.sh -d ...` to get a diff directly, or `-w` to write the changes it proposes automatically.)